hack/verify-godeps.sh

```diff
@@ -101,7 +101,7 @@ pushd "${KUBE_ROOT}" > /dev/null 2>&1
     ret=1
   fi
 
-  if ! _out="$(diff -Naupr -x "BUILD" -x "zz_generated.openapi.go" -x "AUTHORS*" -x "CONTRIBUTORS*" vendor "${_kubetmp}/vendor")"; then
```

Comment:
Won't this affect every file with that name? We have more than one, correct?

Reply:
this is the only one under staging, and if we had others, we'd want to publish them as well (we shouldn't make published staging repos depend on build processes inside k/k)

Reply:
OK. It's not at all obvious that this command is constrained to the staging directory.

If the intent is to get everything in staging (which I agree with) then the path in the other diffs is overly specific.

Reply:
this line (verify-godeps.sh) is undoing https://github.com/kubernetes/kubernetes/pull/75184. The script's intent is to make sure the checked-in staging/vendor code is the same as a fresh `godep restore&save` (with whatever build process running in serial/parallel). https://github.com/kubernetes/kubernetes/pull/75184 added an exception for that uncommitted openapi definitions to make the script pass. We withdraw the exception as we commit that openapi now

the intent for the rest changes is to commit that generated openapi so `go get/build k8s.io/apiextensions-apiserver` doesn't fail #75338.

the diffs were specific because I didn't want to advocate committing more generated files, but we probably have to do the same for sample-apiserver https://github.com/kubernetes/kubernetes/pull/74133 in future. 

> if we had others, we'd want to publish them as well

I agree now. We have to commit all files for the staging repos to be consumed as dependencies.